src/services/cartService.ts

```diff
@@ -0,0 +1,31 @@
+import { CartRepository } from '../repository/cartRepository';
+import { ProductRepository } from '../repository/productRepository';
+import { NotFound, UnprocessableEntity } from '../error/errors';
+
+
+export class CartService {
+  private cartRepository = new CartRepository();
```

Comment:
take a loot at the example of dependency injection using constructor
[https://dev.to/ruben_alapont/the-power-of-dependency-injection-in-typescript-3m5e](https://dev.to/ruben_alapont/the-power-of-dependency-injection-in-typescript-3m5e)